zerver/lib/actions.py

```diff
@@ -1864,6 +1864,22 @@ def do_change_full_name(user_profile, full_name, log=True):
         send_event(dict(type='realm_bot', op='update', bot=payload),
                    bot_owner_userids(user_profile))
 
+def do_change_bot_owner(user_profile, bot_owner, log=True):
+    # type: (UserProfile, UserProfile, bool) -> None
+    user_profile.bot_owner = bot_owner
+    user_profile.save(update_fields=["bot_owner"])
+    if log:
+        log_event({'type': 'user_change_owner',
+                   'user': user_profile.email,
+                   'owner': user_profile.bot_owner.email})
+    send_event(dict(type='realm_bot',
+                    op='update',
+                    bot=dict(email=user_profile.email,
+                             user_id=user_profile.id,
+                             owner=user_profile.bot_owner.email,
```

Comment:
Instead of sending the owner's email, we should send the owner's userid here, and call the field owner_id.